# Partial revert of https://gitlab.freedesktop.org/pipewire/pipewire/-/commit/11320cf20394386a1e686af356a54c2b1154818b
# https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/3622
# https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/3744
# Looks like firefox still needs two buffers.
diff --git a/spa/plugins/audioconvert/audioconvert.c b/spa/plugins/audioconvert/audioconvert.c
index 8dd0da8c62ed7d3f76b95eaa016013b0ba378e76..f0eeff8d99d0bd803ad4208b764171f1e1bbe6e7 100644
--- a/spa/plugins/audioconvert/audioconvert.c
+++ b/spa/plugins/audioconvert/audioconvert.c
@@ -2071,7 +2071,7 @@ impl_node_port_enum_params(void *object, int seq,
 
 		param = spa_pod_builder_add_object(&b,
 			SPA_TYPE_OBJECT_ParamBuffers, id,
-			SPA_PARAM_BUFFERS_buffers, SPA_POD_CHOICE_RANGE_Int(1, 1, MAX_BUFFERS),
+			SPA_PARAM_BUFFERS_buffers, SPA_POD_CHOICE_RANGE_Int(2, 1, MAX_BUFFERS),
 			SPA_PARAM_BUFFERS_blocks,  SPA_POD_Int(port->blocks),
 			SPA_PARAM_BUFFERS_size,    SPA_POD_CHOICE_RANGE_Int(
 								size * port->stride,
